libraries/AP_Beacon/AP_Beacon_Marvelmind.cpp

```diff
@@ -86,20 +83,28 @@ uint8_t AP_Beacon_Marvelmind::mark_position_ready()
     return ind_cur;
 }
 
+uint32_t parse_uint32_le(uint8_t* buffer)
```

Comment:
These type of methods exist already: https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL/utility/sparse-endian.h